File: Backend/PRUEBA .NET_CORE-ANGULAR/PRU.Application/Dtos/Dashboard/Response/DashboardResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PRU.Application.Dtos.Dashboard.Response
{
    public class DashboardResponse
    {
        public int idUsuario { get; set; }
        public string? FechaIngreso { get; set; }
        public string? FechaCierre { get; set; }
        public string? SesionExitosa { get; set; }
        public string? SesionFallida { get; set; }
        public string? SesionActive { get; set; }
        public int? IntentosFallidos { get; set; }
        public string? UserName { get; set; }
        public string? StatusUsuario { get; set; }

    }
    public class DashboardResponseResumen
    {
        public int TotalUsuarios { get; set; }
        public int TotalBloqueados { get; set; }
        public int TotalSesionActiva { get; set; }


    }

}

File: FrontEnd/src/app/pages/Administrador/components/Dashboard/Models/dashboardResponse.ts
export interface Dashboardresponse {
    idUsuario: number
    userName: string
    sesionActive: string
    statusUsuario: string
    fechaIngreso: string
    fechaCierre: Date
    sesionExitosa: string
    sesionFallida: string
    badgeColor:string
    badgeColor2:string
    badgeColor3:string
    badgeColor4:string
    badgeColor5:string
}
export interface DashboardCount {
    totalUsuarios: number
    totalSesionActiva: number
    totalBloqueados: number
}